.coafile

```diff
@@ -4,6 +4,10 @@ bears = SpaceConsistencyBear, LineLengthBear, CoffeeLintBear
 use_spaces = True
 max_line_length = 80
 
+[PYTHON]
+files = **.py
+bears = PEP8Bear
+
```

Comment:
ok, I get it, my bad :sweat:. So that developers can just run coala and get it patched without a prompt :sweat_smile: